docs/source/platform/federation.mdx

```diff
@@ -0,0 +1,228 @@
+---
+title: Federation
+description: Leveraging the Apollo Graph Manager for Federation
+---
+[//]: # (Description: Introductory content, discussing the overall model of the federation platform, why you would want to register partial schemas, and framing the concept of thinking about running federation as a coordination and collaboration problem to motivate the tools we've built)
+[//]: # (Assignee: Chang)
+[//]: # (Reviewer: Caydie, James)
+
+Organizations expose their data as a single graph in order to have a complete view of all of their organizational data along with well-defined and reusable patterns for data access, observability, security, caching, and other API-level considerations. However, even though exposing all of this data in a unified way has [huge benefits](/intro/benefits/), it comes with additional coordination across multiple teams, all of whom own a portion of the data graph. As an organization, ideally individual teams are able to work independently on GraphQL microservices without needing to coordinate changes with other teams. By the same token, you want to ensure that changes from each team are safe for the entirety of the graph and don't violate any assumptions or cause conflicts with other service teams.
```

Comment:
We need to mention the concept of the Apollo Graph Manager here if we're going to follow it up with a section on that manager